packages/domain/src/general/tradeChecklist.ts

```diff
@@ -0,0 +1,66 @@
+import {z} from 'zod'
+import {UnixMilliseconds} from '../utility/UnixMilliseconds.brand'
+
+/**
+ * TODO move to apps/mobile
+ */
+export const TradeChecklistItemStatus = z.enum([
+  /**
+   * Ready to be sent to the other side
+   */
+  'readyToSend',
+  /**
+   * Waiting for response from the other side
```

Comment:
Also 'waiting for my response based on other side input'. Consider adding another status type

Reply:
Not needed for now. Let's see if this is another UI requirement. We can add it then